db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/WindowFunctionNode.java

```diff
@@ -87,6 +87,7 @@ public void init(Object arg1, Object arg2, Object arg3, Object arg4, Object arg5
      * @throws StandardException
      */
     public void init(Object arg1, Object arg2) throws StandardException {
+        super.init();
```

Comment:
I haven't check if such constructor exists, but should we call `super.init(arg1)` instead of `super.init()`?

Reply:
arg1 is used to initialise `this.window` which is `WindowFunctionNode` specific.